sonar-plugin-api/src/main/java/org/sonar/api/measures/FileLinesContextFactory.java

```diff
@@ -32,12 +31,6 @@
 public interface FileLinesContextFactory {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Annotate the "FileLinesContextFactory" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1609)